src/components/list-todo.tsx

```
@@ -9,19 +9,19 @@ import {
   useQueryClient,
 } from "@tanstack/react-query";
 
+import GoalIcon from "@/assets/Goal.svg";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)